src/status_im2/common/confirmation_drawer/style.cljs

```diff
@@ -1,7 +1,7 @@
-(ns status-im.ui2.screens.common.alert.style
+(ns status-im2.common.confirmation-drawer.style
   (:require [quo2.foundations.colors :as colors]))
 
-(defn context-container []
+(def context-container
   {:flex-direction   :row
    :background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
```

Comment:
actually it shouldn't be reactive atome, because we reset the root and re-render the entire tree when we change theme, so we could change it to regular atom to avoid confusion

Reply:
@flexsurfer: Even if `quo2.theme/theme` was a raw `atom` and not reactive, `context-container` is still a static map that would be using the incorrect colors after the atom/ratom changes. I don't understand how using `def` instead of `defn` is related to the reactivity of the state, because the bug would still be the same. If the static map relies on the deref'ed value of the state, then it should be inside a function.